Gruntfile.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * gpii-grunt-mdjson-lint
+ *
+ * Copyright (c) 2017 Raising the Floor Internationals
+ *
+ * Licensed under the BSD-3-Clause license.
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.require("%gpii-grunt-lint-all/src/load-npm-tasks-properly.js");
+module.exports = function (grunt) {
+    // This is required because of the inherent circularity of this setup.
```

Comment:
Could you list more clearly here what the circle is and what modules it passes through

Reply:
Done.